Userland/Tests/Kernel/TestKernelPledge.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2018-2021, the SerenityOS developers.
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibTest/TestCase.h>
+#include <unistd.h>
+
+TEST_CASE(test_nonexistent_pledge)
+{
+    auto res = pledge("testing123", "notthere");
+    if (res >= 0) {
+        FAIL("Pledging on existent promises should fail.");
+    }
```

Comment:
since we have a single-line body in all of these `if`s now, let's drop the curlies :)

```suggestion
    if (res >= 0)
        FAIL("Pledging on existent promises should fail.");
```